### ISSUE TITLE
Published Jakarta EE 8 BOM misses jakarta.xml.bind-api and jakarta.xml.ws-api

### ISSUE THREAD

**Issue by bogdandrutu** [jakartaee/jakartaee-api#136]
This issue was fixed in https://github.com/jakartaee/jakartaee-api/pull/56, but not ported to 8.0.0 BOM.

**Comment by scottkurz** [comment]
Noting that the discussion in https://github.com/jakartaee/jakartaee-api/issues/97 seemed to assume that all the right changes had been made in 8.0.x-BRANCH, and the only thing needed was to publish a release.  

But this issue suggests that some of the other additions in #56 need to be merged into the 8.0.x BOM as well.   E.g. the jakarta.xml.bind-api dependency is already included in https://github.com/jakartaee/jakartaee-api/blob/8.0.x-BRANCH/jakartaee-bom/pom.xml but the jakarta.xml.ws-api is not. 

I haven't personally reviewed what the contents of the 8.0.x BOM should be but hopefully it was helpful for me to capture that summary.

I know @ivargrimstad had attempted to do a release but hit a problem with the build infrastructure, so mentioning him for when he has a chance to look here.

**Comment by bogdandrutu** [comment]
@scottkurz you are correct, we should also add `jakarta.xml.ws-api` to the bom. My rule of thumb is that we should add all artifacts that the repo produces in the bom.

**Comment by scottkurz** [comment]
> @scottkurz @bogdandrutu If one were to backport the changes in #56 applicable to 8.0.x, would that give some impetus to release 8.0.1 to resolve #97 and and #136 (this issue)? I am curious whether it would be wasted time if no one is committed to a 8.0.1 release.

@ivargrimstad do you have a sense of if/when you might make another attempt at an 8.0.x update release?  It's possible we might see another PR proposal to build into it here.   Thanks.

**Comment by ivargrimstad** [comment]
> @ivargrimstad do you have a sense of if/when you might make another attempt at an 8.0.x update release? It's possible we might see another PR proposal to build into it here. Thanks.

I don't have any plans for pursuing this at the moment. Any committer on the Platform Project can take the initiative to get it done. You may bring it up on the next platform call to gauge interest. If anyone wants to look at the job and see if they can get it to work, feel free to give it a try. https://ci.eclipse.org/jakartaee-platform/job/ONLY_FOR_EE8-StagingRelease/

**Comment by yeroc** [comment]
I guess there's no chance this gets resolved?
